api/models.py

```diff
@@ -162,6 +162,9 @@ class Node(DatabaseModel):
     name: str = Field(
         description='Name of the node object'
     )
+    group: Optional[str] = Field(
+        descriptoin='Name of a group this node belongs to'
```

Comment:
Need to fix a minor typo.
```suggestion
        description='Name of a group this node belongs to'
```